lib/system/generic/generic_io.c

```diff
@@ -15,7 +15,7 @@ void metal_sys_io_mem_map(struct metal_io_region *io)
 {
 	unsigned long p;
 	size_t psize;
-	void *va;
+	char *va;
```

Comment:
This looks like a separate commit

Reply:
Yes, missing it, i will send it in a separate patch